src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double secondLeg, double firstLeg) {
+        super(color);
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }
+
+    public double getFirstLeg() {
+        return firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: rightTriangle, area: "
+                + String.format("%.2f",getArea()) + " sq. units, firstLeg: "
+                + String.format("%.2f",getFirstLeg()) + " units, b: "
+                + String.format("%.2f",getSecondLeg()) + " units, color: " + getColor();
```

Comment:
Instead of using `getColor()`, you should use `color.name()` to get the String representation of the enum constant for the color.